main.py

```diff
@@ -38,23 +46,39 @@ def sentiment_analysis():
     print(f"The lowest sentiment score is {headlines.min_score}")
 
     buy, sell = headlines.get_trade_decisions()
-    orders = helpers.place_trades(buy, sell)
-    helpers.activate_order(orders)
-
-    # sleep for 3 days
-    time.sleep(259200)
+    order_ids = helpers.place_trades(buy, sell)
+    helpers.activate_order(order_ids=order_ids)
 
 
-def perform_sentiment_analysis():
-    lemon_api: LemonMarketsAPI = LemonMarketsAPI()
-    helpers: Helpers = Helpers(lemon_api)
+def schedule_trades_for_year():
```

Comment:
Same comments here as previous update - remove this function, do a venue-open check in sentiment_analysis() and directly schedule the sentiment_analysis() method.